app/src/main/java/com/github/gogiffy/fragments/GifGridFragment.java

```diff
@@ -88,6 +88,11 @@ public void onLoadMore(int page, int totalItemsCount) {
         loadTrendingGifs(0);
     }
 
+    private void unUsedMethod(String args){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "unUsedMethod" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "args". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1172)